packages/lit-html/src/lit-html.ts

```diff
@@ -22,6 +22,19 @@ const wrap =
     ? window.ShadyDOM!.wrap
     : (node: Node) => node;
 
+const trustedTypes = (globalThis as unknown as  Partial<Window>).trustedTypes;
```

Comment:
did this actually save any bytes?

Reply:
Yes before gzip, no difference after gzip